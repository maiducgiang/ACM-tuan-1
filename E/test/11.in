100
552743844 60521081
815592223 441662533
406209717 740367217
5182915 444081655
528142380 330701132
157294238 625014503
777495161 857363259
750384628 466414237
639597945 752881715
611758904 503789125
815381926 203940526
778248613 673470193
968928005 205370356
706495064 3303859
941190961 292660624
874421246 681679291
305254013 448665092
833698648 453667561
673107803 184544949
378871230 377252951
769565310 938659787
141756959 921588390
920894896 572414680
874089974 957596700
644496449 188467428
587199385 844503661
252373732 148929458
494749353 215775209
660997753 608442716
365672319 438748542
646241274 507900086
615888484 60534252
846902691 888445357
340594201 50083287
608445484 833029960
685840799 12292407
92741692 319991954
100195648 909466283
204899178 342826735
179389779 766791533
582886795 178796677
341307354 254299907
573195445 300798253
666818568 443361624
642236359 388229780
922708064 153542028
592694876 61468425
5758181 583054746
823731891 490032776
103523183 951792534
888141290 674253481
511616616 284117475
234782309 971314327
762187004 341350584
404909397 380936935
163449558 725318404
741528629 648346913
187765172 17981399
35203332 786557009
540613446 623896753
735391909 456741018
58355633 696984406
59610383 335563613
978185053 454580267
105084811 737408216
734289351 756355167
273473507 523596082
384062009 786554195
168264550 152458586
238026115 741302858
97237719 347872563
300042171 231738106
175734693 356825926
159721963 946695543
417621479 255390616
903666039 302146929
949878806 591308352
256801561 251516089
308312333 79247263
717045436 62914868
140891880 604635021
457483303 781398781
508265479 831687342
556781426 813460061
284004219 206868739
835003356 661204948
371549997 554411119
709732135 78657970
642821061 764824807
165967196 112006923
159057766 200754707
615790112 713560245
226708694 199764705
747383561 916000357
925824901 48709810
815724137 225011928
796276355 864760276
755278794 625089170
658549775 801909539
413067883 462115873
