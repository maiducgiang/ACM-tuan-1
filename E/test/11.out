2
2
2
86
2
4
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
7
2
2
2
2
2
2
2
2
2
2
2
2
2
2
4
10
2
5
2
2
2
2
2
2
2
102
2
10
2
2
5
2
2
5
2
2
23
2
2
12
6
2
8
2
2
3
2
4
4
2
3
6
2
2
2
2
2
2
5
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
2
